x-pack/plugins/stack_connectors/common/sentinelone/constants.ts

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Can you add the `sentinelone` directories to the `CODEOWNERS` file, like we did with the gen_ai connector?  https://github.com/elastic/kibana/blob/a42df25d4ad87d91e5a0bfd981a082590e27ae0c/.github/CODEOWNERS#L1143-L1146

That way, we'll only get pinged for review when the framework code changes ...